### ISSUE TITLE
SBL SBG Aria-Label

### ISSUE THREAD

**Issue by ghost** [Dental-Revenue/Codebase-2-Dev#168]
The Static Blocks List and Static Block Grid modules should have an aria-label added to their links.
`<a aria-label="An image for * <?php echo $block['block_title']; ?>" href="<?php echo $block['block_url']; ?>" class="static_block_grid-block-image" style="background-image: url(<?php echo $block['block_image']; ?>);"></a>`
